servers/rendering/renderer_rd/forward_mobile/scene_shader_forward_mobile.cpp

```diff
@@ -698,6 +698,11 @@ void SceneShaderForwardMobile::init(const String p_defines) {
 		actions.render_mode_defines["debug_shadow_splits"] = "#define DEBUG_DRAW_PSSM_SPLITS\n";
 		actions.render_mode_defines["fog_disabled"] = "#define FOG_DISABLED\n";
 
+		bool specular_occlusion = GLOBAL_GET("rendering/reflections/specular_occlusion/enabled");
+		if (!specular_occlusion) {
```

Comment:
Same comment as the others, note that this would need to be flipped if you choose to keep the check